src/App.css

```diff
@@ -49,28 +78,89 @@
 
 .nav-menu a {
   text-decoration: none;
-  color: #000;
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: var(--nav-menu-theme);
 }
 
 .nav-menu a:hover {
-  text-decoration: underline;
+  text-decoration: none;
+}
+
+.person-icon {
+  align-items: center;
+}
+
+.oval {
+  position: relative;
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  border: solid 1px var(--white-three);
+}
+
+.mask {
+  position: absolute;
+  width: 1.063rem;
+  height: 1.063rem;
+  border-radius: 50%;
+  background-image: url("../public/person-icon.png");
+  background-size: cover;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
 }
 
+/* nav menu END */
+
+/* main container */
+
 .container {
   display: flex;
```

Comment:
- [ ] Please use reusable classes for reoccurring styles. For example `display: flex:`  is repeated 18 times in this file. It would be cleaner to create a class in CSS with the style and then use that className on all elements that are affected by it in the components. Kindly do so.

e.g
```
.d-flex {
  display:flex;
}
```

```
<div className="d-flex">
</div>

Reply:
Thank you @Munsa1 I will make the suggested implementation to resolve the repeating classes and values.